src/Zoomable.js

```diff
@@ -253,8 +253,8 @@ function getZoom(element) {
   let totalOffset = margin * 2;
   let viewport = getViewportDimensions();
 
-  let scaleX = viewport.width / (getWidth(element) + totalOffset);
-  let scaleY = viewport.height / (getHeight(element) + totalOffset);
+  let scaleX = Math.min(viewport.width / (getWidth(element) + totalOffset), element.dataset.actualWidth / getWidth(element));
+  let scaleY = Math.min(viewport.height / (getHeight(element) + totalOffset), element.dataset.actualHeight / getHeight(element));
```

Comment:
I could see this being more necessary the smaller the image is. Most of what I've used this package for hasn't really had this problem though.